src/js/utils/menu.js

```diff
@@ -29,42 +29,44 @@ export const updateMenuLinks = () => {
 
 export const setMenuState = () => {
   const $navMenu = $(".nav-menu__links");
-  const $signinLink = $(".nav-menu__links a:nth-child(1)");
-  const $registerLink = $(".nav-menu__links a:nth-child(2)");
+  const $loginLink = getAnchorElement(
+    "/accounts/login/",
+    "nav-link w-inline-block",
+    "Login"
+  );
+  const $logoutButton = $("<button />", {
+    class: "nav-link w-inline-block",
+    id: "my-muni-logout",
+    type: "button",
+    text: "Logout",
+  });
+  const $registerLink = getAnchorElement(
+    "/accounts/register/",
+    "nav-link w-inline-block",
+    "Register"
+  );
+  const $settingsLink = getAnchorElement(
+    "/accounts/settings/",
+    "nav-link w-inline-block",
+    "Settings"
+  );
 
-  let $logoutButton = $("#my-muni-logout");
+  $navMenu.find(".nav-link").remove();
+  $navMenu.append([$loginLink, $logoutButton, $registerLink, $settingsLink]);
```

Comment:
This should make auto-close on click work

```suggestion
  $navMenu.append([$loginLink, $logoutButton, $registerLink, $settingsLink]);
  Webflow.require('ix2').init();
```